py_gtfs_rt_ingestion/lib/convert_gtfs.py

```diff
@@ -68,6 +68,7 @@ def convert(self) -> None:
                             s3_prefix,
                         ),
                         partition_cols=["timestamp"],
+                        visitor_func=self.send_metadata,
```

Comment:
have to pass our `visitor_func` to be used as a `file_visitor` in `write_dataset`